### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by flourish03**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by flourish03 — COMMENTED** [review]
the idea of pull request is awesome

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
   // Initialization
```
Comment (flourish03) [inline index.html:53]: index.html changes is nice

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (flourish03) [inline index.html:47]: no much changes

**Comment by flourish03** [comment]
its good

File: README.md
Comment (flourish03) [inline README.md:1]: i love git pull

**Review by flourish03 — COMMENTED** [review]
i love git pull

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Michael2004-ukpeh) [inline index.html:51]: Need documentation

**Review by Michael2004-ukpeh — COMMENTED** [review]
Reviewed

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (Michael2004-ukpeh) [inline index.html]:
```suggestion
`<p> A paragraph</p>`
```

Comment (Michael2004-ukpeh) [reply index.html]: A suggestion

**Review by Michael2004-ukpeh — COMMENTED** [review]
Reviewed

File: index.html [hunk 5]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
```
Comment (Michael2004-ukpeh) [inline index.html:48]:
```suggestion
</table>
```

**Pull request merged**